pinot-server/src/main/java/org/apache/pinot/server/api/resources/MmapDebugResource.java

```diff
@@ -65,11 +75,10 @@ public List<String> getOffHeapSizes() {
   @GET
   @Produces(MediaType.APPLICATION_JSON)
   @Path("/memory/offheap/table/{tableName}")
-  @ApiOperation(value = "Show off heap memory consumed by latest mutable segment",
-      notes = "Returns off heap memory consumed by latest consuming segment of realtime table")
+  @ApiOperation(value = "Show off heap memory consumed by latest mutable segment", notes = "Returns off heap memory "
+      + "consumed by latest consuming segment of realtime table")
   @ApiResponses(value = {
-      @ApiResponse(code = 200, message = "Success"),
-      @ApiResponse(code = 500, message = "Internal server error"),
+      @ApiResponse(code = 200, message = "Success"), @ApiResponse(code = 500, message = "Internal server error"),
```

Comment:
should this have addl swagger annotations? also, unrelated formatting

Reply:
The Swagger annotation is added to the class instead of each endpoints.
 The format change is good catch.